Sources/PrimerSDK/Classes/User Interface/PaymentMethodsGroupView.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  PaymentMethodsGroupView.swift
+//  PrimerSDK
+//
+//  Created by Evangelos Pittas on 4/10/21.
+//
+
+import UIKit
+
+protocol PaymentMethodsGroupViewDelegate {
+    func paymentMethodsGroupView(_ paymentMethodsGroupView: PaymentMethodsGroupView, paymentMethodTapped paymentMethod: PaymentMethodViewModel)
+}
+
+class PaymentMethodsGroupView: PrimerView {
+    
+    internal private(set) var title: String?
+    internal private(set) var paymentMethodsViewModels: [PaymentMethodViewModel]!
+    private var stackView: UIStackView = UIStackView()
+    internal var delegate: PaymentMethodsGroupViewDelegate?
+    internal var titleLabel: UILabel?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        render()
+    }
+    
+    convenience init(frame: CGRect, title: String?, paymentMethodsViewModels: [PaymentMethodViewModel]) {
+        self.init(frame: frame)
+        self.title = title
+        self.paymentMethodsViewModels = paymentMethodsViewModels
+        render()
+    }
+    
+    func render() {
+        backgroundColor = .black.withAlphaComponent(0.05)
+        translatesAutoresizingMaskIntoConstraints = false
+        layer.cornerRadius = 4.0
+        clipsToBounds = true
+        
+        addSubview(stackView)
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.alignment = .fill
+        stackView.distribution = .fill
+        stackView.spacing = 7.0
+        stackView.pin(view: self, leading: 10, top: 10, trailing: -10, bottom: -10)
+        
+        let theme: PrimerThemeProtocol = DependencyContainer.resolve()
+        
+        if let title = title {
+            titleLabel = UILabel()
+            titleLabel!.text = title
+            titleLabel!.textAlignment = .right
+            titleLabel!.textColor = theme.colorTheme.text1
+            stackView.addArrangedSubview(titleLabel!)
+        }
+        
+        for paymentMethodViewModel in paymentMethodsViewModels {
+            let paymentMethodButtonView = PaymentMethodButtonView(frame: .zero, viewModel: paymentMethodViewModel)
+            paymentMethodButtonView.backgroundColor = .clear
+            paymentMethodButtonView.titleLabel?.font = UIFont.systemFont(ofSize: 17.0, weight: .medium)
+            paymentMethodButtonView.cornerRadius = 4.0
+            paymentMethodButtonView.borderWidth = 1.0
+            paymentMethodButtonView.borderColor = .clear
+            paymentMethodButtonView.clipsToBounds = true
+            
+            switch paymentMethodViewModel.type {
+            case .paymentCard:
+                paymentMethodButtonView.setTitleColor(theme.colorTheme.text1, for: .normal)
+                paymentMethodButtonView.tintColor = theme.colorTheme.text1
+                paymentMethodButtonView.buttonColor = .white
```

Comment:
we need to discuss taking all these colors and placing them in the theme or similar